src/rockstor/scripts/config_backup.py

```diff
@@ -0,0 +1,29 @@
+"""
+Copyright (c) 2012-2016 RockStor, Inc. <http://rockstor.com>
+This file is part of RockStor.
+
+RockStor is free software; you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published
+by the Free Software Foundation; either version 2 of the License,
+or (at your option) any later version.
+
+RockStor is distributed in the hope that it will be useful, but
+WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program. If not, see <http://www.gnu.org/licenses/>.
+"""
+
+import os
+os.environ['DJANGO_SETTINGS_MODULE'] = 'settings'
+from system.config_backup import backup_config
+
+def main():
+    result = backup_config()
+    print('config exported to: "%s"' % result.filename)
```

Comment:
@daniel-illi - @schakrava 

Check my comment on pr, move `result.filename` to `result['filename']`

Mirko